### ISSUE TITLE
$only_save_to_taxonomy still stores empty values in post meta

### ISSUE THREAD

**Issue by mboynes** [alleyinteractive/wordpress-fieldmanager#361]
Given the following code:

```php
function my_fm_terms() {
	$fm = new Fieldmanager_Group( array(
		'name' => 'terms',
		'children' => array(
			'categories' => new Fieldmanager_Select( array(
				'label' => __( 'Category', 'fm-test' ),
				'remove_default_meta_boxes' => true,
				'datasource' => new Fieldmanager_Datasource_Term( array(
					'taxonomy' => 'category',
					'only_save_to_taxonomy' => true,
				) ),
			) ),
			'tags' => new Fieldmanager_Select( array(
				'label' => __( 'Tag', 'fm-test' ),
				'remove_default_meta_boxes' => true,
				'datasource' => new Fieldmanager_Datasource_Term( array(
					'taxonomy' => 'post_tag',
					'only_save_to_taxonomy' => true,
				) ),
			) ),
		),
	) );
	$fm->add_meta_box( __( 'Terms', 'fm-test' ), array( 'post' ) );
}
add_action( 'fm_post_post', 'my_fm_terms' );
```

When I add terms, they correctly save the term relationships. However, Fieldmanager will in this case also store the post meta `a:2:{s:10:"categories";a:0:{}s:4:"tags";a:0:{}}`. It should automatically set `$skip_save` on a field if the datasource is overriding the save. Note that this depends on #360 to work properly.

**Comment by dlh01** [comment]
Duplicate of #270?
